.travis.yml

```diff
@@ -1,16 +1,9 @@
 language: ruby
 matrix:
   allow_failures:
-    - rvm: jruby-head
     - rvm: ruby-head
 rvm:
-  - jruby-19mode
-  - jruby-head
-  - 1.9.3
-  - 2.0.0
-  - 2.1.5
-  - 2.1.6
-  - 2.2.2
```

Comment:
My comment got killed with the browser dying !!
Anyways, the answer is: no, so  I can add it back in